src/View/PdfView.php

```diff
@@ -130,6 +130,14 @@ public function render(?string $template = null, false|string|null $layout = nul
         return $this->Blocks->get('content');
     }
 
+    /**
+     * @inheritDoc
+     */
+    public function getConfig(?string $key = null, mixed $default = null): mixed
```

Comment:
Cake 5 makes the base method protected, but it's used in the unit tests. I opened it again, but i'm also open to other solutions.

Reply:
The protected visibility is a mistake in the core. It will be fixed in `5.0.1`.

Reply:
I guess we'll wait for te release of 5.0.1 here, then.

Reply:
This class also needs

	/**
	 * Mime-type this view class renders as.
	 *
	 * @return string Either the content type or '' which means no type.
	 */
	public static function contentType(): string {
		return '...;
	}
	
It seems.

Reply:
With this fragment, the `testRenderErrorTemplate` fails and i'm wondering, if that test actually makes sense. I think i do not know enough about Cakes Error handling to judge that.

Reply:
I'll look into it.